src/site/paragraphs/table.drupal.liquid

```diff
@@ -0,0 +1,61 @@
+{% comment %}
+    Example data:
+    entity: {
+        "entityId": "3944",
+        "fieldTable": {
+            "tableValue": "Row 1 Column 1 Row 1 Column 2 Row 1 Column 3 Row 2 Column 1 Row 2 Column 2 Row 2 Column 3 Row 3 Column 1 Row 3 Column 2 Row 3 Column 3",
+            "value": {
+                "0": [
+                "Row 1 Column 1",
+                "Row 1 Column 2",
+                "Row 1 Column 3"
+                ],
+                "1": [
+                "Row 2 Column 1",
+                "Row 2 Column 2",
+                "Row 2 Column 3"
+                ],
+                "2": [
+                "Row 3 Column 1",
+                "Row 3 Column 2",
+                "Row 3 Column 3"
+                ],
+            }
+            "format": "rich_text",
+            "caption": "Test Table"
+        }
+    }
+{% endcomment %}
+
+
+<table class="va-table">
+
+    {% if entity.fieldTable.caption %}
+        <caption>{{ entity.fieldTable.caption }}</caption>
```

Comment:
May want to get clarity from @kevwalsh on this. I think he said `caption` would be visually hidden.

Reply:
I can take out - altho I noticed tables usually come paired with H3s that seem to function similarly to captions. Where do those get migrated to?

Reply:
At the moment, to captions. I think I got that wrong though. I'm going to confirm with Kev before taking it out. After that, nothing will be migrated to captions.

Reply:
Kev says you're right. I won't migrate the paired H3s into caption, but anything graphql finds in caption does belong in `<caption>`. So the above code is correct.